1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature

```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
         And the user is informed that progress can be viwed in My Messages 
 
     @needJiraTag
-    Scenario Outline: Owner wants to publish Resource, all restrictions
-        Given Owner wants to publish their Resource
-        And Resource is of type "<ResourceType>" 
-        And Editor has restricted publication of type "<ResourceType>" 
+    Scenario: Owner wants to publish Resource, all restrictions
```

Comment:
In this file, I feel we need to have consistent capitalisation of NVA concepts. Resource, maybe Published Resource and Landing page